templates/compose/rybbit.yaml

```
@@ -0,0 +1,138 @@
+# documentation: https://rybbit.io/docs
+# slogan: Open-source, privacy-first web analytics.
+# tags: analytics,web,privacy,self-hosted,clickhouse,postgres
+# logo: svgs/rybbit.svg
+# port: 3000
+
+services:
+  rybbit:
+    image: 'ghcr.io/rybbit-io/rybbit-client:latest'
+    container_name: rybbit-client
+    environment:
+      - NODE_ENV=production
+      - 'NEXT_PUBLIC_BACKEND_URL=${SERVICE_URL_RYBBIT}'
+      - 'NEXT_PUBLIC_DISABLE_SIGNUP=${DISABLE_SIGNUP:-false}'
```

The `port: 3000` mentioned on the comments at the top of the compose is useful for users who are using `caddy` as their Coolify Proxy but those who use `traefik` as their Coolify Proxy might see `no available server` error because of the missing `SERVICE_URL_RYBBIT_3000`

The `SERVICE_URL_RYBBIT_3000` also tells Coolify to automatically generate a sub domain for the service with port `3000` at the end so the user just have to change the domain to the domain they prefer to use but the auto generated domain will work out of the box (if they have DNS configured properly to resolve all of their sub domains to the server where they are deploying this service)

```suggestion
    environment:
      - SERVICE_URL_RYBBIT_3000
      - NODE_ENV=production
      - 'NEXT_PUBLIC_BACKEND_URL=${SERVICE_URL_RYBBIT}'
      - 'NEXT_PUBLIC_DISABLE_SIGNUP=${DISABLE_SIGNUP:-false}'
```